html/includes/authenticate.inc.php

```diff
@@ -19,13 +21,13 @@
 // Clear up any old sessions
 dbDelete('session', '`session_expiry` <  ?', array(time()));
 
-\Delight\Cookie\Session::start('Strict');
+Session::start('Strict');
 
 $time = time() - 60 * 60 * 24 * $config['auth_remember'];
-setcookie('sess_id', '', $time, '/', null, $config['secure_cookies']);
-setcookie('PHPSESSID', '', $time, '/', null, $config['secure_cookies']);
-setcookie('token', '', $time, '/', null, $config['secure_cookies']);
-setcookie('auth', '', $time, '/', null, $config['secure_cookies']);
+Cookie::setcookie('sess_id', '', $time, '/', null, $config['secure_cookies'], true, 'Strict');
```

Comment:
The old setcookie was in here to clear them out, without doing this then the old cookie / session info keeps you logged in and you can't logout.

Reply:
I don't understand why I would have made this change.